arionBackend/transformation/transformator.py

```diff
@@ -0,0 +1,8 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Transformer(metaclass=ABCMeta):

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a docstring to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1720)